2-mandatory/9-cheap-diner.js

```diff
@@ -19,8 +19,23 @@ chosenMeal(emptyArray)
 Should give the answer "Nothing :("
 **/
 
-function chooseMeal(mealArray) {}
+function chooseMeal(mealArray) {
+  
+  let lowestPriceMealArray = (mealArray.map(x=>x.price).sort((a,b) => a-b))
```

Comment:
you could also sort on the parameter. I think you wanted to cover from the empty array cases but you could have this statement after :
```
 if (mealArray.length==0){
    return "Nothing :("
  }
  ```
  so that you are sure you will always have at least a param

Reply:
thank you for this advice